src/index.jsx

```diff
@@ -86,14 +127,17 @@ const App = () => (
           openSourceUrl={process.env.OPEN_SOURCE_URL}
           termsOfServiceUrl={process.env.TERMS_OF_SERVICE_URL}
           privacyPolicyUrl={process.env.PRIVACY_POLICY_URL}
-          facebookUrl={process.env.FACEBOOK_URL}
-          twitterUrl={process.env.TWITTER_URL}
-          youTubeUrl={process.env.YOU_TUBE_URL}
-          linkedInUrl={process.env.LINKED_IN_URL}
-          googlePlusUrl={process.env.GOOGLE_PLUS_URL}
-          redditUrl={process.env.REDDIT_URL}
           appleAppStoreUrl={process.env.APPLE_APP_STORE_URL}
           googlePlayUrl={process.env.GOOGLE_PLAY_URL}
+          socialLinks={socialLinks}
+          enterpriseMarketingLink={{
+            url: process.env.ENTERPRISE_MARKETING_URL,
```

Comment:
Do we need to add these to the local configuration and also our Stage and Prod configs?

Reply:
@christopappas and I have created an edx-internal PR for adding these configs on stage/prod for all the MFEs we're touching.

Good call on the local configuration. Missed that in this repo; adding it now.

Reply:
yeah, at this point, all the relevant stage and prod config variables and values have been merged to internal

Reply:
👍